spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+require 'pry'
+
+
+describe UsersController do
+
+  let(:user){create(:user)}
+
+  context '#create' do
+
+    it 'should create a new User with proper parameters' do
+      expect{post :create, user: attributes_for(:user)}.to change(User, :count).by(1)
+    end
+
+    it 'should not create a new User without all parameters' do
+      expect{post :create, user: attributes_for(:user, first_name: nil)}.to change(User, :count).by(0)
+    end
+  end
+
+  context 'logged in user' do
```

Comment:
I did not think about testing redirecting the user from the login page to the show or timeline page.